elements/lisk-p2p/src/p2p.ts

```diff
@@ -640,8 +640,30 @@ export class P2P extends EventEmitter {
 
 	private async _startPeerServer(): Promise<void> {
 		this._scServer.on(
-			'connection',
+			'handshake',
 			(socket: SCServerSocket): void => {
+				// Terminate the connection the moment it receive ping frame
+				(socket as any).socket.on('ping', () => {
```

Comment:
wouldn't this happen on the normal scenario? smme for `pong`

Reply:
These low level `ping` messages are not supposed to be sent by lisk nodes so it should be fine to disconnect with them as soon as we receive it.